terraform/main.tf

```diff
@@ -76,22 +76,26 @@ module "batch_tile_cache_image" {
 
 
 module "fargate_autoscaling" {
-  source                    = "git::https://github.com/wri/gfw-terraform-modules.git//terraform/modules/fargate_autoscaling?ref=v0.4.2.3"
-  project                   = local.project
-  name_suffix               = local.name_suffix
-  tags                      = local.fargate_tags
-  vpc_id                    = data.terraform_remote_state.core.outputs.vpc_id
-  private_subnet_ids        = data.terraform_remote_state.core.outputs.private_subnet_ids
-  public_subnet_ids         = data.terraform_remote_state.core.outputs.public_subnet_ids
-  container_name            = var.container_name
-  container_port            = var.container_port
-  desired_count             = var.desired_count
-  fargate_cpu               = var.fargate_cpu
-  fargate_memory            = var.fargate_memory
-  auto_scaling_cooldown     = var.auto_scaling_cooldown
-  auto_scaling_max_capacity = var.auto_scaling_max_capacity
-  auto_scaling_max_cpu_util = var.auto_scaling_max_cpu_util
-  auto_scaling_min_capacity = var.auto_scaling_min_capacity
+  # source                       = "git::https://github.com/wri/gfw-terraform-modules.git//terraform/modules/fargate_autoscaling?ref=single-dev-lb"
```

Comment:
Temporarily pointing to local module as that that repo reads an update to enable this. There's a [PR](https://github.com/wri/gfw-terraform-modules/pull/15/files?diff=split&w=1) there that I'll point this to before merging once that's released.